crates/ty_python_semantic/resources/mdtest/attributes.md

```diff
@@ -1904,6 +1904,7 @@ we only consider the attribute assignment to be valid if the assigned attribute
 from typing import Literal
 
 class Date:
+    # error: [invalid-method-override]
```

Comment:
Not necessary to decide in this PR, but I wonder if we should special-case ignoring `object.__setattr__` for Liskov purposes (and separately implement validation of a correct `__setattr__` signature). TBH I'm not even sure why typeshed has `object.__setattr__`, and I don't think there is any soundness need for subclasses of `object` to respect Liskov compatibility with the `object.__setattr__` definition in typeshed.